build/style-gutenburg.css

```diff
@@ -6493,8 +6489,7 @@ Block: Staggered Box (Frontend styles)
   .edit-post-visual-editor .mojblocks-staggered-box__image,
   .block-editor-block-styles .mojblocks-staggered-box__image,
   .block-editor-inserter__menu-help-panel .mojblocks-staggered-box__image {
-    -webkit-filter: invert(100%);
-            filter: invert(100%);
+    filter: invert(100%);
```

Comment:
Looks good so far :) how come some of the -webkit-filter things have been removed?

Reply:
No, idea, this file gets compiled by Webpack. For this commit I've not done any CSS changes so it's funny something has changed there. I'll take a look and see if I can see anything that might have changed the compile.

Reply:
I wonder if someone removed this but then didn't compile the CSS until I've come along and done it? Anyway, I've added it back in so hopefully that's what we want. It didn't seem to cause issue on my local machine.